ods_ci/run_robot_test.sh

```diff
@@ -317,19 +329,30 @@ echo "${exit_status}"
 
 # send test artifacts by email
 if ${EMAIL_REPORT}
- then
-     tar cvzf rf_results.tar.gz ${TEST_ARTIFACT_DIR} &> /dev/null
+  then
+     tar cvzf rf_results.tar.gz "${TEST_ARTIFACT_DIR}" &> /dev/null
```

Comment:
was it failing?

Reply:
I think he was just fixing shellcheck warning about word splitting https://github.com/koalaman/shellcheck/wiki/SC2046